lapce-core/queries/svelte/highlights.scm

```diff
@@ -0,0 +1,109 @@
+; src: https://github.com/helix-editor/helix/blob/master/runtime/queries/svelte/highlights.scm
+; licence: https://github.com/helix-editor/helix/blob/master/LICENSE
+
```

Comment:
could it be consistent with other queries and include spdx?

Reply:
added it